corehq/apps/export/tests/test_export_conversion.py

```diff
@@ -130,6 +133,24 @@ def test_nested_repeat_conversion(self):
         self.assertEqual(column.label, 'Modified Nested')
         self.assertEqual(column.selected, True)
 
+#    def test_system_property_conversion(self):
+#        saved_export_schema = SavedExportSchema.wrap(self.get_json('system_properties'))
+#        with mock.patch(
```

Comment:
here i'm mocking the schema and then generating the instance from that. because of that, it has no system properties. once system props are columns, this'll just work since the columns will be injected during instance creation